### PR TITLE
Use Stoplight Elements to render OpenAPI docs

### PR DESCRIPTION
**Pull request by carlhoerberg**
### WHAT is this pull request doing?

Instead of compiling static html files from `openapi.yaml` using `redoc` the docs are now rendered in the browser using https://github.com/stoplightio/elements

### HOW can this pull request be tested?

Compile and then browse to http://localhost:15672/docs/

### CODE REVIEW

**Comment by carlhoerberg** [comment]
Required (or a variant of it) by #692. Installing nodejs with a recent version in all distros (ubuntu 20.04, debian 11) is a hassle, and just required for building the static html.

File: static/docs/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>LavinMQ HTTP API</title>
+    <script src="https://unpkg.com/@stoplight/elements/web-components.min.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/@stoplight/elements/styles.min.css">
```
Comment (dentarg) [inline static/docs/index.html:8]: It is probably a good idea lock these to a certain version so they does not break on us unexpectedly. Using Subresource Integrity (SRI) we can also ensure that it is the correct version served from the CDN: https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

**Review by snichme — APPROVED** [review]
Looks good!

File: static/docs/index.html (hunk 1 above)
Comment (carlhoerberg) [reply static/docs/index.html:8]: or download locally?

Comment (carlhoerberg) [reply static/docs/index.html]: https://github.com/cloudamqp/lavinmq/pull/693/commits/e91b966906ab4ac3f28329edd91e106fa7579ccc

**Pull request merged**